pkg/resourceinterpreter/defaultinterpreter/default.go

```diff
@@ -106,3 +112,23 @@ func (e *DefaultInterpreter) GetDependencies(object *unstructured.Unstructured)
 	}
 	return handler(object)
 }
+
+// ReflectStatus returns the cluster object's running status after the grab.
+func (e *DefaultInterpreter) ReflectStatus(object *unstructured.Unstructured) (status *runtime.RawExtension, err error) {
+	handler, exist := e.reflectStatusHandlers[object.GroupVersionKind()]
+	if exist {
+		return handler(object)
+	}
+
+	// for no default build-in resource kind, directly collect the entire status.
+	statusMap, _, err := unstructured.NestedMap(object.Object, "status")
```

Comment:
Please add a unit test for this function. I want to see what happened if no `.status` field.
Maybe we can use the second return value to suppress error logs.

Reply:
Done.